server/auth/authorization.ts

```diff
@@ -1,18 +1,18 @@
+export {};
 const express = require('express');
-const app = express();
-const jwt = require('jsonwebtoken');
 const dotenv = require('dotenv');
+const jwt = require('jsonwebtoken');
 
-
-let refreshTokens = [];
+const app = express();
+let refreshTokens: any = [];
 app.use(express.json());
 dotenv.config();
 
-function auth(req, res, next){
+function auth(req: any, res: any, next: any){
```

Comment:
The request/response body can be made with a DTO object to have a more clear type of object, Typescript helps very well with that. 
Please use strictly the object and needed fields that should be used. 
`any` type is a very unknown thing and you should write clear types. Thank you :)